python/unitytrainers/ppo/trainer.py

```diff
@@ -95,8 +95,18 @@ def __init__(self, sess, env, brain_name, trainer_parameters, training, seed):
 
         self.summary_writer = tf.summary.FileWriter(self.summary_path)
 
+        self.inference_run_list = [self.model.output, self.model.all_probs, self.model.value,
```

Comment:
Can the same run_list system be implemented in BC ?

Reply:
Just did it.